data/json/items/melee/unarmed_weapons.json

```
@@ -99,21 +47,67 @@
     "flags": [ "UNARMED_WEAPON" ]
   },
   {
-    "type": "GENERIC",
-    "id": "punch_dagger",
-    "name": { "str": "punch dagger" },
-    "description": "A short and sharp double-edged dagger made to be gripped in the palm, with the blade protruding between the fingers.",
-    "weight": "168 g",
-    "to_hit": -1,
-    "price_postapoc": 50,
-    "color": "dark_gray",
-    "symbol": "{",
-    "material": [ "steel" ],
+    "id": "gloves_studded",
+    "type": "ARMOR",
+    "name": { "str": "pair of studded gloves", "str_pl": "pairs of studded gloves" },
+    "description": "A pair of gloves with studded metal knuckles.",
+    "weight": "218 g",
     "volume": "250 ml",
+    "price": 1100,
+    "price_postapoc": 250,
+    "bashing": 2,
+    "cutting": 1,
+    "material": [ "steel", "leather", "cotton" ],
+    "symbol": "[",
+    "looks_like": "gloves_leather",
+    "color": "light_gray",
+    "covers": [ "hand_l", "hand_r" ],
+    "coverage": 100,
+    "encumbrance": 5,
+    "warmth": 5,
+    "material_thickness": 2,
+    "flags": [ "WATER_FRIENDLY", "STAB", "DURABLE_MELEE" ]
+  },
+  {
+    "id": "knuckle_impact",
+    "type": "ARMOR",
+    "name": { "str": "impact knuckles" },
+    "description": "A pair of thick steel plates covering your proximal phalange. They are strapped to your hands with metal braces.",
```

```suggestion
    "description": "A pair of thick steel plates covering your proximal phalange.  They are strapped to your hands with metal braces.",
```

You just need to click "accept change"